### DISCUSSION TITLE
Edit patterns?

### DISCUSSION THREAD

**Discussion by smrki1** [danielmiessler/Fabric#128] [Q&A] [answered]
Hey all, is there a simple way to edit Patterns? I have just noticed extract_wisdom gives repetitive answer in sense that the same idea/sentence is found in more than one section of the answer.

**Comment by smrki1** [comment, marked as answer]
Ah I see, patterns are stored in your fabric folder. Easy.

**Comment by danielmiessler** [comment]
Precisely.
